src/app/importers/import_spreadsheet.py

```diff
@@ -329,8 +338,8 @@ async def spikes(
     suffixes = list(set(suffixes))
 
     for i in suffixes:
-        spike_name: str = sample_import[f"spike_name_{i}"]
-        spike_quantity: str = sample_import[f"spike_quantity_{i}"]
+        spike_name: str = sample_import.get(f"spike_name_{i}", "")
+        spike_quantity: str = sample_import.get(f"spike_quantity_{i}", "")
 
         if is_none_or_nan(spike_name) or is_none_or_nan(spike_quantity):
```

Comment:
Without fully understanding the logic, should this also check if `spike_name == "" or spike_quantity == ""`? Or just default to `None` from the `.get()`?